src/controllers/userController.js

```diff
@@ -0,0 +1,90 @@
+import { save, getById, update, getAutoSuggestUsers, getAllUsers, remove } from '../services/UserService';
+import { v4 as uuidv4 } from 'uuid';
+import { bad } from '../util/constant';
+import logger from '../util/logger';
+
+export const saveUser = async (request, response) => {
+    const userId = uuidv4();
+    const userLogin = request.body.login;
+    const userPassword = request.body.password;
+    const userAge = request.body.age;
+    const isUserDeleted = false;
+    logger.info(`saveUser {id: ${userId}, login: ${userLogin}, password: ${userPassword}, age: ${userAge}, isDeleted: ${isUserDeleted}}`);
+    try {
+        const user = await save(
+            { id: userId, login: userLogin, password: userPassword, age: userAge, isDeleted: isUserDeleted });
+        response.status(201).send(user);
+    } catch (err) {
+        logger.error(`saveUser failed: {id: ${userId}, login: ${userLogin}, password: ${userPassword}, age: ${userAge}, isDeleted: ${isUserDeleted}}, message: ${err.message}`);
+        response.status(500).send(bad);
```

Comment:
Just note. It would be better to return json instead of plain text. in case of json body it is much easier to parse it on client side. Also it is preferable to use same format/type for success response(in your case it is json) and for fail responses

Reply:
Yes. I changed it.